specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2021-06-01/examples/DeleteApplication.json

```diff
@@ -12,7 +12,11 @@
         "location": "https://management.azure.com/subscriptions/subid/providers/Microsoft.HDInsight/pathToOperationResult"
       }
     },
-    "202": {},
+    "202": {
+      "headers": {
+        "location": "https://management.azure.com/subscriptions/subid/providers/Microsoft.HDInsight/pathToOperationResult"
```

Comment:
Could you please provide one real location URL format to make "pathToOperationResult" more concrete?
PS: We can replace the concrete operation id with empty guide.

Reply:
The same with the below

Reply:
Updated location URL, but not provide "pathToOperationResult", this path is randomly generated. The  location value the users get is return by RP, I think "pathToOperationResult" is better than real path to understand in examples.

![image](https://user-images.githubusercontent.com/58678302/222386450-217b4c9f-4056-4408-86a8-0c6a35a3c3d2.png)

Reply:
Done